<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>flight_inquiry_citys_and_date</class>
 <widget class="QWidget" name="flight_inquiry_citys_and_date">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1031</width>
    <height>665</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>70</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Yu Gothic UI Semilight</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QLabel" name="UserStatus">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>70</y>
     <width>241</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Yu Gothic UI Semilight</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QTableView" name="Flights">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>1000</width>
     <height>500</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Yu Gothic UI Semilight</family>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Yu Gothic UI Semilight</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Flight Infomation</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>821</width>
     <height>61</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <family>Yu Gothic UI Semilight</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Departure City</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="DepatureCity">
      <property name="font">
       <font>
        <family>Yu Gothic UI Semilight</family>
        <pointsize>11</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="font">
       <font>
        <family>Yu Gothic UI Semilight</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Arrival City</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="ArrivalCity">
      <property name="font">
       <font>
        <family>Yu Gothic UI Semilight</family>
        <pointsize>11</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="font">
       <font>
        <family>Yu Gothic UI Semilight</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Departure Date</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDateEdit" name="dateEdit">
      <property name="font">
       <font>
        <family>Yu Gothic UI Semilight</family>
        <pointsize>11</pointsize>
       </font>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>70</y>
     <width>120</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Yu Gothic UI Semilight</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>120</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Yu Gothic UI Semilight</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Confirm</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>70</y>
     <width>120</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Yu Gothic UI Semilight</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
